boards/arm/nrf7002dk_nrf5340/Kconfig.defconfig

```diff
@@ -71,6 +71,25 @@ config FLASH_LOAD_SIZE
 
 endif # BOARD_NRF7002DK_NRF5340_CPUAPP_NS
 
+if NETWORKING
```

Comment:
We should do this for nRF7000/nRF7001 also.

Reply:
Yes, definitely.

Reply:
@krish2718  Now looking at this, I would add a similar defconfig for nR7001 DK/shield, but what about nRF7000 shield? `WPA_SUPP` is not needed for sure, but do `NETWORKING`/`NET_L2_ETHERNET` need to be enabled for scan only? Or is `WIFI`/`WIFI_NRF700X` enough?

Reply:
`WIFI_NRF700X` and `NET_L2_ETHERNET` both are needed.